src/Persistence/MassTransit.EntityFrameworkCoreIntegration/MsSqlLockStatements.cs

```diff
@@ -26,18 +26,15 @@ public class MsSqlLockStatements : IRawSqlLockStatements
 
```

Comment:
Damn github PR isn't letting me put a comment on an "unchanged" line. Anyways, Line 19 is a using statement that is unused. So it can be removed.